plugins/modules/zos_copy.py

```diff
@@ -151,6 +151,22 @@
     type: bool
     default: false
     required: false
+  force_lock:
+    description:
+      - By default, when c(dest) is a MVS data set and is being used by another
+        process with DISP=SHR or DISP=OLD the module will fail. Use C(force_lock)
+        to bypass this check and continue with copy.
+      - If set to C(true) and destination is a MVS data set opened by another
+        process then zos_copy will try to copy using DISP=SHR.
+      - Using C(force_lock) uses operations that are subject to race conditions
+        and can lead to data loss, use with caution.
+      - If a dataset member has aliases, and is not a program
+        object, copying that member to a dataset that is in use will result in
```

Comment:
This is minor, but `dataset` should be `data set` , couple of places in the `force_lock` description.

Not a necessary change and if you update, i don't think you need to run regression.

Reply:
Agree, I copied this from ZOAU docs and didn't pay attention